lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3395,10 +3395,12 @@ namespace Js
 #if ENABLE_NATIVE_CODEGEN
         Assert(!IsIntermediateCodeGenThunk(entryPoint));
 #endif
+#ifdef ENABLE_SCRIPT_PROFILING
```

Comment:
Is this needed? It looks like this entire function is already under ENABLE_SCRIPT_PROFILING?